app/src/main/res/layout/fragment_search_filter.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:id="@+id/rootView"
+    android:orientation="vertical"
+    tools:context="org.fossasia.openevent.general.search.SearchFilterFragment">
+
+    <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <TextView
+            android:id="@+id/PriceLabel"
+            android:padding="@dimen/padding_medium"
+            android:textSize="@dimen/text_size_expanded_title"
+            android:text="@string/price"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content" />
+
+        <LinearLayout
+            android:orientation="horizontal"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+            <TextView
```

Comment:
why a textview checkbox already have text property to display text no need to add linearlayout and textview just add a checkbox with text @string/free_stuff would be fine

Reply:
I would have to use extra `android:LayoutDirection=rtl` for to cheive the same layout. If that would be fine. Then its okay. Thats why i did not use checkBox text

Reply:
but it is better than adding extra views and nesting of views

Reply:
Yeah I understand. It was in heat of the moment I tried to quickly convert it.  Changed it now. Skipped my mind